docs/components/modeler/web-modeler/connectors/available-connectors/slack.md

```diff
@@ -0,0 +1,92 @@
+---
+id: slack
+title: Slack connector
+description: Working with Slack connector
+---
+
+The **Slack connector** allows you to send messages to channels or users in your [Slack](https://slack.com) workspace from your BPMN process.
+
+## Prerequisites
+
+To use the Slack connector, a Slack app must be registered with the Slack workspace you would like to send messages to. A respective OAuth token needs to be configured as a secret in your cluster. Follow [these steps in the appendix](#appendix) to learn how to set this up.
+
+## Create a Slack connector task
+
+To use a **Slack connector** in your process, either change the type of an existing task using the **change type** context menu, or create a new connector task by using the **append** context menu. Follow [our guide on using connectors](../use-connectors.md) to learn more.
+
+## Make your Slack connector executable
```

Comment:
Personal opinion (non-blocking): I'd think about the wording _make executable_.

What we're doing here from my point of view is _to configure it_ (so it runs). (You model it, you configure it ==> applies to all connectors).



> "highlighted in red" -> This _should_ be obvious given the UI + label, right? I'd personally not repeat it in this case.

Reply:
Makes sense. But I will ditch it for now, because this is the term we use with all connectors. Before making a decision, I would like to reflect in general how to best set-up our connector docs.